membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -58,13 +59,75 @@ object AccountHelpers {
 
 }
 
+case class CancellationEffectiveDate(cancellationEffectiveDate: String)
```

Comment:
```suggestion
case class CancellationEffectiveDate(cancellationEffectiveDate: LocalDate)
```

Reply:
This is by design to avoid MMA having to check if the date represents `now`. So MDAPI will respond with string that is either `"now"` or `"yyyy-mm-dd"`, latter representing the end of invoice date.

Reply:
👍